### PR TITLE
Cards component added.

### LINKED ISSUE #5: Implement the Category Cards for the Landing Page
The design for the the cards is given in the Figma design which can be accessed via the notion link provided in the readme.
The images for the cards used in the design is provided in the public folder.

### PR DESCRIPTION
**Pull request by NightFury742**
fixes #5 

-created CardData.mjs  `(src/config/CardData.mjs)`
-modified content.jsx  `(pages/LandingPage/content.jsx)` to render cards.
-modified CategoryCards.jsx (components/cards/CategoryCards.jsx) to add cards.
-added **underline.svg** to public folder .
-made the Cards component responsive upto 350px using tailwind breakpoints.

|Screenshot from 2023-06-02 11-37-26 |Screenshot from 2023-06-02 11-37-34|
| ------------- | ------------- |

Screenshot from 2023-06-02 11-49-53

### CODE REVIEW

**Comment by NightFury742** [comment]
@sambhavgupta0705 @Anurag607  I have made the card component as requested, please review it.
Thanks

**Comment by sambhavgupta0705** [comment]
Hey @NightFury742 make the title a little small

**Comment by Anurag607** [comment]
@NightFury742 make sure to pull from the master branch

**Comment by NightFury742** [comment]
@sambhavgupta0705 @Anurag607 I have made the changes as requested, please review them.
Thanks

File: src/pages/LandingPage/index.jsx [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
+import Content from './content';
+import Footer from '../../components/Footer';
 
 const LandingPage = () => {
   return (
     <>
-        <NavBar />
-        <HeroSection />
-        <Content />
-        <Footer />
```
Comment (sambhavgupta0705) [inline src/pages/LandingPage/index.jsx:13]: I think you use prettier.
Try avoiding it while submitting a PR.

**Comment by sambhavgupta0705** [comment]
@NightFury742 Please share a preview

**Comment by NightFury742** [comment]
Screenshot from 2023-06-04 23-40-45
Screenshot from 2023-06-04 23-39-32

File: src/pages/LandingPage/content.jsx [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
 const Content = () => {
+  const card = CardData.map((value) => {
+    return <CategoryCards key={value.id} card={value} />;
+  });
   return (
-    <div>Content</div>
-  )
-}
+    <div className="text-center">
+      <p className="comfort pl-4 pr-4  m-0 max-sm:text-[2.3rem]  max-sm:tracking-[-0.2rem]  max-sm:leading-[40px]  font-bold text-[4rem] leading-[5rem]  text-[#0f172A] tracking-[-0.5rem]">
```
Comment (sambhavgupta0705) [inline src/pages/LandingPage/content.jsx:10]: use lg: and try it making a little big for bigger screen

**Comment by NightFury742** [comment]
Added breakpoint for 'lg' as requested.
Screenshot from 2023-06-05 03-14-48
Screenshot from 2023-06-05 03-14-36
Screenshot from 2023-06-05 03-14-23

**Review by sambhavgupta0705 — APPROVED** [review]
Nice work @NightFury742 
@Anurag607 PTAL once , everything is fine from my side.

**Comment by Anurag607** [comment]
image
as you can see...on 1280px resolution the content is way too big, make it a bit smaller if possible.

also try to make the changes I requested earlier.

**Comment by sambhavgupta0705** [comment]
@NightFury742

**Comment by NightFury742** [comment]
I am sorry @sambhavgupta0705 , I was busy with my quizzes and lab tests, I will definitely resolve this issue by today.

**Comment by NightFury742** [comment]
@Anurag607 @sambhavgupta0705 I have fixed the responsiveness and box shadow as well, to make the cards appear as they were in the figma file.
Please review my changes.
Screenshot from 2023-06-11 20-40-33
Screenshot from 2023-06-11 20-41-01
Screenshot from 2023-06-11 20-41-29
Screenshot from 2023-06-11 20-42-04

**Comment by sambhavgupta0705** [comment]
Well Done @NightFury742

**Pull request merged**